2025REDOBuildSeason/src/main/java/frc/robot/Auto/Actions/TurnDegreesAction.java

```diff
@@ -0,0 +1,81 @@
+package frc.robot.Auto.Actions;
+
+//imported subsystems that help calculate turn
+import frc.robot.Subsystem.Mecanum;
+import frc.robot.Devices.Gyro;
+
+// imported math things
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+
+/* 
+ * This action turns the robot a certain number of degrees
+ * for a certain number of seconds
+ * negative is left and positive is right
+ * May immplement pid at some point
```

Comment:
```suggestion
 * May implement pid at some point
```
Is this done? This part of the comment isn't helpful